src/histogram.rs

```diff
@@ -873,6 +876,89 @@ impl LocalHistogram {
     }
 }
 
+///delegator for auto flush-able local counter
+pub trait AFLHistogramDelegator<T: 'static + MayFlush> {
```

Comment:
Is it possible to just reuse `AFLDelegator` but add an associated type for its metric type?

Reply:
> Is it possible to just reuse `AFLDelegator` but add an associated type for its metric type?

Afraid not, because GenericLocalCounter and LocalHistogram have different number of type parameters.